### ISSUE TITLE
Use the "Display Entity" for holograms

### ISSUE THREAD

**Issue by SnowCutieOwO** [DecentSoftware-eu/DecentHolograms#160]
### Just making sure

- [X] I have read the wiki and made sure, this feature is not yet in the plugin.
- [x] I believe, that this feature is possible to implement.
- [X] I made sure, that this feature has not been requested yet.

### Problem

Like Fancy Hologram plugin, this entity introduced in new version could reduce the usage of memory, and have a better performance.

### Feature

Use Display Entity in 1.19+ of Minecraft.
Could be disabled in config.yml

### Alternatives

May could be done by using dh's entity hologram but I think this is not the best solution.

**Comment by Andre601** [comment]
Duplicate of #120

I'll quote my reply from there:
> We're well aware about the Display Entities that will be introduced in the upcoming MC release.
> 
> This will bring a lot of cool features, but at the same time a lot of work, especially when d0by wants to keep backwards compatability with the older MC releases, which obviously do not have these features.
> 
> As an example, the entities as of right now can...
> - ...have their text rotation locked
> - ...have the bg disabled
> - ...have custom bg color
> - ...have custom text size
> - ...have text alignment
> 
> Those are only a few of the features and they all wouldn't be available in older versions (Or at least not without heavy work), so a separate setup would be needed to be made here to achieve support while keeping compat.
> 
> Other, smaller issues and inconveniences are things like how to support existing features of DH. For example, how should per-line permissions to hide/show them be supported with the new entity, if people also want to use text alignment? It would either require splitting up the lines into individual entities (Which is a waste of its line-break support) or add some logic to split the lines that do have special things like line height (Which would also cause issues), permission, etc.
> 
> tl;dr: We know about that feature, we want to support it, but it will take a lot of work and time to support it, so patience is apreciated.
